feathr-impl/src/test/java/com/linkedin/feathr/common/AutoTensorizableTypesTest.java

```diff
@@ -6,15 +6,14 @@
 import com.linkedin.feathr.common.types.PrimitiveType;
 import java.util.Optional;
 
-import org.scalatest.testng.TestNGSuite;
 import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
 
 import static java.util.Collections.*;
 import static org.testng.Assert.*;
 
 
-public class AutoTensorizableTypesTest extends TestNGSuite {
+public class AutoTensorizableTypesTest {
```

Comment:
Seems like there is a bunch of scala test related changed, can you update the description to reflect why the changes were made?